src/model/Global.ts

```diff
@@ -119,11 +121,13 @@ class GlobalModel extends Model<ECUnitOption> {
         parentModel: Model,
         ecModel: GlobalModel,
         theme: object,
+        locale: object,
         optionManager: OptionManager
     ): void {
         theme = theme || {};
         this.option = null; // Mark as not initialized.
         this._theme = new Model(theme);
+        this._locale = locale;
```

Comment:
I think we should create a new model for `locale`, just like `theme`. 
That is what @pissang suggested in https://github.com/apache/incubator-echarts/pull/13038#discussion_r462027492 and using model may reduce some redundant logic of fetching deeply.